jobs/build_docker/prepare_docker_post_test.sh

```diff
@@ -2,12 +2,17 @@
 set -x
 set -e
 
+
 # Stop host mongo and rabbitmq service
 # otherwise mongo and rabbitmq service won't run normally
 # use host mongo and rabbitmq is a chioce but the coverage may be decreased.
 # services will be restart in cleanup.sh, which script will always be executed.
-echo $SUDO_PASSWORD |sudo -S service mongodb stop
-echo $SUDO_PASSWORD |sudo -S service rabbitmq-server stop
+if [ "$(service mongodb         status|grep  start )" != "" ]; then
+    echo $SUDO_PASSWORD |sudo -S service mongodb stop
+fi
+if [ "$(service rabbitmq-server status|grep  start )" != "" ]; then
```

Comment:
I try to get the status of rabbitmq, and get:
```
xxx@ubuntuxxx:~$ service rabbitmq-server status
erlexec: HOME must be set

Only root or rabbitmq should run rabbitmqctl
```
I also tried to get the status of rabbitmq with sudo and get ""

Maybe we can check the status of rabbitmq by:
```
if  [ "$(sudo service rabbitmq-server status|grep  pid)" != "" ]; then
    ......
else
   ......
fi
```